### PR TITLE
Fix typo (privledge->privilege)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by apasel422** [comment]
@bors: r+ rollup

Thanks!

**Comment by steveklabnik** [comment]
Gah! Thanks!

@bors: r+ rollup

File: src/libstd/env.rs [hunk 1]
```diff
@@ -526,7 +526,7 @@ pub fn temp_dir() -> PathBuf {
 /// Ok("/home/alex/bar")
 /// ```
 ///
-/// This sort of behavior has been known to [lead to privledge escalation] when
+/// This sort of behavior has been known to [lead to privilege escalation] when
 /// used incorrectly, for example.
 ///
 /// [lead to privledge escalation]: http://securityvulns.com/Wdocument183.html
```
Comment (apasel422) [inline src/libstd/env.rs]: Could you update this too, so that the link still works?

**Comment by steveklabnik** [comment]
@bors: r-

whoops, @apasel422 i will let you have this PR, I didn't see that. lol!

**Comment by apasel422** [comment]
@qolop Are you still interested in fixing this?

**Comment by qolop** [comment]
@apasel422 Sorry about that, I have just fixed the second typo.

**Comment by apasel422** [comment]
@bors: r+ rollup

Thanks @qolop!

**Pull request merged**